e2e-tests/tests/infra/appSections/AddPageSection.ts

```diff
@@ -1,4 +1,9 @@
 import { Locator, Page, expect } from "@playwright/test";
+import { StreamScope } from "@yext/studio-plugin";
+
+type StreamScopeForm = {
+  [key in keyof StreamScope]: string;
```

Comment:
I guess we don't use this quite yet, but would we want the values to be `string[]`?

Reply:
Updated!